src/slack/slack-events-handler.service.ts

```diff
@@ -40,6 +46,10 @@ export class SlackEventsHandlerService {
   private handleEventCallback(eventBody: EventCallbackEvent) {
     const innerEvent = eventBody.event,
       teamId = eventBody.team_id;
+    this.logger.log('📨  Slack event_callback received', {
```

Comment:
Let's not add emoji to the logs
```suggestion
    this.logger.log('Slack event_callback received', {
```

Reply:
Sure, I have made the change to not include emoji to the logs.